template/app/buy-me-coffee/index.tsx

```diff
@@ -5,6 +5,7 @@ import Banner from '../../src/components/banner/banner';
 import Header from '../../src/components/header/Header';
 import BuyMeCoffeeContractDemo from '../../src/pageComponents/buy-me-coffee/ContractDemo';
 import Guide from '../../src/pageComponents/buy-me-coffee/Guide';
+import Introduction from '../../src/pageComponents/buy-me-coffee/Introduction';
```

Comment:
I wonder if we should have a different name that is not introduction 🤔

@ppiyush13 @arsood what do you think?

Reply:
I usually struggle with naming variables :D 
How about `Profile` or `Bio` ?

Reply:
I like `Profile`

Reply:
Done